lib/benchee/system.ex

```diff
@@ -139,6 +144,11 @@ defmodule Benchee.System do
 
   def parse_cpu_for(:FreeBSD, raw_output), do: String.trim(raw_output)
 
+  def parse_cpu_for(:Solaris, raw_output) do
+    {:ok, [decoded]} = Jason.decode(raw_output)
```

Comment:
This works by accident. Benchee does not depend on `Jason` and should not depend on it. It's available in the dev and test environments due to dev/test dependencies:

```
tobi@qiqi:~/github/benchee(main)$ mix deps.tree
benchee
├── credo ~> 1.7.2-rc.0 (Hex package)
│   ├── bunt ~> 0.2.1 or ~> 1.0 (Hex package)
│   ├── file_system ~> 0.2 or ~> 1.0 (Hex package)
│   └── jason ~> 1.0 (Hex package)
├── deep_merge ~> 1.0 (Hex package)
├── dialyxir ~> 1.0 (Hex package)
│   └── erlex >= 0.2.6 (Hex package)
├── ex_doc >= 0.0.0 (Hex package)
│   ├── earmark_parser ~> 1.4.39 (Hex package)
│   ├── makeup_elixir ~> 0.14 (Hex package)
│   │   ├── makeup ~> 1.0 (Hex package)
│   │   │   └── nimble_parsec ~> 1.2.2 or ~> 1.3 (Hex package)
│   │   └── nimble_parsec ~> 1.2.3 or ~> 1.3 (Hex package)
│   └── makeup_erlang ~> 0.1 (Hex package)
│       └── makeup ~> 1.0 (Hex package)
├── ex_guard ~> 1.3 (Hex package)
│   └── fs ~> 8.6.1 (Hex package)
├── excoveralls ~> 0.13 (Hex package)
│   └── jason ~> 1.0 (Hex package)
├── statistex ~> 1.0 (Hex package)
└── table ~> 0.1.0 (Hex package)
```

so, we need make do here without `Jason` :)